app/templates/organizations/organization/change-events/new.hbs

```diff
@@ -304,7 +304,7 @@
                     {{else}}
                       <div class={{if hasError "ember-power-select--error"}}>
                         <PowerSelect
-                          @options={{@model.changeEvent.originalOrganizations}}
+                          @options={{has-many-value @model.changeEvent "originalOrganizations"}}
```

Comment:
I went with a new helper since we might need it in other places that I haven't encountered yet.

I decided not to go for EmberData's `Await` component or ember-promise-helpers since the data is already loaded, we simply want sync access to the already loaded data.